sign-lib/src/main/java/ee/ria/DigiDoc/sign/SignLib.java

```diff
@@ -100,8 +100,20 @@ public static void overrideProxy(String host, int port, String username, String
         }
     }
 
+    private static String getSchemaPath(Context context) {
+        return getSchemaDir(context).getAbsolutePath();
+    }
+
+    private static void initHomeDir(Context context) {
+        String path = getSchemaPath(context);
+        try {
+            Os.setenv("HOME", path, true);
+        } catch (ErrnoException e) {
+            Timber.log(Log.ERROR, e, "Setting HOME environment variable failed");
+        }
+    }
+
     private static void initNativeLibs() {
-        System.loadLibrary("c++_shared");
```

Comment:
You can remove the binary files from repo

Reply:
Done